data_utils/serialized_dataset_loader.py

```diff
@@ -21,16 +24,12 @@ class SerializedDataLoader:
 
     Methods
     -------
-    load_serialized_data(dataset_path: str, config: dict)
+    load_serialized_data(dataset_path: str, config: dict, verbosity: int)
```

Comment:
Let's make this a class member to slightly reduce how much we have to pass this around

Reply:
@streeve 
I just pushed the new version with your suggestion